serverless/develop/serverless-event-delivery.adoc

```diff
@@ -6,9 +6,15 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can configure event delivery parameters for Knative Eventing that are applied in cases where an event fails to be delivered by a xref:../../serverless/develop/serverless-subs.adoc#serverless-subs[subscription] or xref:../../serverless/develop/serverless-triggers.adoc#serverless-triggers[trigger] to a subscriber. Event delivery parameters are configured individually per subscriber.
+You can configure event delivery parameters that are applied in cases where an event fails to be delivered to an event sink. If an event fails to be delivered, the error code shown is a standard HTTP error code.
```

Comment:
> If an event fails to be delivered, the error code shown is a standard HTTP error code.

shown where? The sentence would make sense in the context of sending events (as an application developer, sending events to a Broker or Channel, you'd get an HTTP error code if the event was not sent, but once it was accepted by the Broker/Channel, you would not see any error even if an eventual Sink doesn't accept the event, as the actual delivery to sinks is decoupled from the process of sending them)

So, I think in the context of event _delivery_  , the sentence is kind of backwards. If the event sink responds with an HTTP error code, the event delivery is treated as a failure (and then the `retries` and `Dead letter sink` configuration applies, depending on whether the error code is retry-able or not (see https://github.com/knative/specs/blob/main/specs/eventing/data-plane.md#event-acknowledgement-and-delivery-retry ) .  So, the sink would show the error code to whatever was dispatching the event to the sink (usually, a kafka broker/channel dispatcher), but that is not anything the user or it's application can see.

Reply:
I can just cut this sentence entirely if it makes sense to do so?

It was only added after discussions (IIRC w/ Matthias) about not needing to show the error codes, which had been previously requested and therefore documented, so I removed the section about error codes and instead just added the statement about HTTP error codes. It sounds like it might be fine to just leave this out entirely though.

Thanks for the detailed explanation, that is very helpful 🙏🏻 🙂

Reply:
yes, I think having just

> You can configure event delivery parameters that are applied in cases where an event fails to be delivered to an event sink.

would be OK